contracts/football_bets.py

```
@@ -40,37 +127,67 @@ def get_match_result() -> str:
+        except Exception:
+            raise Exception("Invalid JSON response from match resolver")
+
+        if not isinstance(result_json, dict):
+            raise Exception("Invalid match result format")
+
+        if "winner" not in result_json or "score" not in result_json:
+            raise Exception("Missing required fields in match result")
+
+        try:
+            winner = int(result_json["winner"])
+        except:
```

Ruff (0.15.7)

[error] Do not use bare `except`

(E722)